admin/brand.php

```diff
@@ -155,12 +96,8 @@
     </div>
 
     <form method="POST" action="brand.php">
```

Comment:
When we submit the form, we probably want to add the brand id as a parameter to the action. If we don't add it, when the form is submitted how will the brand.php page know which brand we're editing?